examples/int8/training/vgg16/main.py

```diff
@@ -167,6 +173,50 @@ def main():
                 ckpt_dir=args.ckpt_dir,
             )
 
+    if args.fp8_epochs > 0:
```

Comment:
Let's move this into a separate file. 
1) Create a vgg16_fp8_ptq file in `examples/dynamo` directory ([reference](https://github.com/pytorch/TensorRT/blob/68a5892021d9d273c8cbb67777d664bd2b55f716/examples/dynamo/torch_compile_stable_diffusion.py))

2) This script should instantiate a model and load regular checkpoint weights. ([example](https://github.com/pytorch/TensorRT/blob/c21866bcb84b3cb3212ce4007a6dbad8196d2c31/examples/int8/training/vgg16/export_ckpt.py#L43-L58))
3) Then perform the `model mtq.quantize(model, quant_cfg, forward_loop=calibrate_loop)` with the calibrate loop loading examples from from training dataset.
4) Print out the accuracy at the end. 
The final demo should be rendered similar to https://pytorch.org/TensorRT/tutorials/_rendered_examples/dynamo/torch_compile_resnet_example.html
So the training code will stay in `examples/int8/training/vgg16` directory while the PTQ demo will be in `examples/dynamo/`